src/index.tsx

```diff
@@ -76,42 +76,46 @@ export function PangolinProvider({
   children,
   account,
   theme,
+  mixpanelToken,
 }: {
   chainId: number | undefined;
   library: any | undefined;
   account: string | undefined;
   children?: React.ReactNode;
   theme?: any;
+  mixpanelToken?: string;
```

Comment:
I think we need to add this into env instead of as a prop because if someone is using components then they will not pass mixpanel in that case we will not be able to track events. please confirm with Justin